tests/app/dao/test_services_dao.py

```diff
@@ -884,60 +887,21 @@ def test_update_service_creates_a_history_record_with_current_data(
     service_cleanup([service.id], notify_db_session.session)
 
 
-@pytest.mark.serial  # Need to run in serial to ensure nothing weird gets added
 def test_create_service_and_history_is_transactional(
     notify_db_session,
     sample_user,
 ):
+    service_id = uuid.uuid4()
     user = sample_user()
     service = Service(name=None, email_from='email_from', message_limit=1000, restricted=False, created_by=user)
 
     with pytest.raises(IntegrityError) as excinfo:
-        dao_create_service(service, user)
+        dao_create_service(service, user, service_id)
 
     ServiceHistory = Service.get_history_model()
     assert 'column "name" of relation "services_history" violates not-null constraint' in str(excinfo.value)
-    assert notify_db_session.session.scalars(select(Service)).all() == []
-    assert notify_db_session.session.scalars(select(ServiceHistory)).all() == []
-
-
-@pytest.mark.serial
```

Comment:
The method this is testing is `delete_service_and_all_associated_db_objects`, and the only place it is used is within commands.py which is slated for removal and currently unused.